src/bmeg/emitter.py

```diff
@@ -127,6 +128,7 @@ def emit_edge(self, obj: Edge, from_gid: GID, to_gid: GID):
     @enforce_types
     def emit_vertex(self, obj: Vertex):
         dumped = {
+            "_id": obj.gid(),
```

Comment:
The engine changes the `_id` to `gid` on the fly when processing. The next line (`"gid": obj.gid()`) will be ignored by the engine, so it should be removed.